packages/rum/src/viewCollection.ts

```diff
@@ -26,26 +30,31 @@ export interface ViewMeasures {
   userActionCount: number
 }
 
+export enum ViewLoadType {
+  INITIAL_LOAD = 'initial load',
+  ROUTE_CHANGE = 'route change',
```

Comment:
are spaces well supported in the search? should not use _ to be more consistent with other types?